pkg/rehearse/jobs.go

```diff
@@ -301,9 +308,11 @@ func (e *Executor) waitForJobs(jobs sets.String, selector string) (bool, error)
 	}
 }
 
-func (e *Executor) submitRehearsals() (sets.String, error) {
+func (e *Executor) submitRehearsals() (sets.String, []*pjapi.ProwJob, error) {
 	var errors []error
-	pjs := make(sets.String, len(e.rehearsals))
+	pj_names := make(sets.String, len(e.rehearsals))
```

Comment:
Prefer `pjNames` over `pj_names`

Reply:
I know you didn't change this, but `len(e.rehearsals)` is the worst case scenario for the size of this set, right? Why do we need to do something other than use the `sets.NewStrings()` constructor?

Reply:
Addressed.